cat/CaT-service.yaml

```diff
@@ -19,10 +19,34 @@ paths:
     put:
       tags:
         - Profile Management
-      description: Registers an existing Conclave user to use Tenders API.
+      description: Registers an existing Conclave user to use Tenders API or updates the existing record with the latest data in Conclave.
       operationId: app.registerUser
 
       responses:
+        '200':
+          description: Success (for object update).
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  userAction:
```

Comment:
Is `userAction` redundant now that a `200` response represents a successful update and a `201` a successful create?